meta/terminal42/contao-mp_forms/de.yml

```
@@ -0,0 +1,7 @@
+de:
+    title: Mehrseitige Formulare
+    description: >
+    		Diese Erweiterung ermöglicht es dir ein mehrseitiges Formular zu erstellen.
```

Tabs and spaces are mixed in this line. Please use spaces only:

```suggestion
        Diese Erweiterung ermöglicht es dir ein mehrseitiges Formular zu erstellen.
```